webapp/src/Components/Player/Overview/SideBar/PlayerStatsCard.tsx

```diff
@@ -58,28 +71,59 @@ class PlayerStatsCardComponent extends React.PureComponent<Props, State> {
-                                    favourite car
-                                </Typography>
-                            </Grid>
-                            <Grid item xs={3} container direction="column" alignItems="center">
-                                <Grid item>
-                                    <Typography align="center">
-                                        {this.state.playerStats.car.carName}
+                        <>
+                            <Grid container alignItems="center" justify="space-around" spacing={8}>
+                                <Grid item xs={3}>
+                                    <DirectionsCar/>
+                                </Grid>
+                                <Grid item xs={6}>
+                                    <Typography variant="subheading">
+                                        favourite car
                                     </Typography>
                                 </Grid>
-                                <Grid item>
-                                    <Typography className={classes.percentage}>
-                                        {roundNumberToMaxDP(this.state.playerStats.car.carPercentage * 100, 1)}%
+                                <Grid item xs={3} container direction="column" alignItems="center">
+                                    <Grid item>
+                                        <Typography align="center">
+                                            {this.state.playerStats.car.carName}
+                                        </Typography>
+                                    </Grid>
+                                    <Grid item>
+                                        <Typography className={classes.percentage}>
+                                            {roundNumberToMaxDP(this.state.playerStats.car.carPercentage * 100, 1)}%
+                                        </Typography>
+                                    </Grid>
+                                </Grid>
+                            </Grid>
+                            <Grid container alignItems="center" justify="space-around" spacing={8}>
+                                <Grid item xs={3}>
+                                    <People/>
+                                </Grid>
+                                <Grid item xs={9}>
+                                    <Typography variant="subheading">
+                                        favourite players
                                     </Typography>
                                 </Grid>
                             </Grid>
-                        </Grid>
+
+                            <Grid container alignItems="center" justify="space-around" spacing={8}>
+                                <Grid item xs={12}>
+                                    <List component="nav">
+                                        {this.state.playerStats.playersInCommon.map((person) =>
+                                            <>
```

Comment:
don't need the `<> </>` as it's a single thing, you also need `key={person.id}` i think.